yaml/env_configuration.go

```diff
@@ -31,6 +31,8 @@ func ParseEnvConfiguration(data []byte) (*EnvConfiguration, error) {
 	return r, nil
 }
 
+// EnvConfigurationFromWorkspaceConfig is used to generate an IaC configuration from types.WorkspaceConfig
```

Comment:
Does this need addressed before merging?

Reply:
I added this note before I started working on capability names and the full effect of this PR.
I'm going to drop this deprecation notice because it's slowly working toward a more-desirable output for users as we introduce a better data model. (i.e. we're preserving the user intent in the config so we can spit *that* out to yaml)